### DISCUSSION TITLE
Modworkshop API and BeardLib updates

### DISCUSSION THREAD

**Discussion by MrDogit** [diesel-modding/PAYDAY-2-BeardLib#590] [Ideas]
As I understand it, modworkshop changed its API about a month ago. The problem is that BeardLib(4.8 and lower) itself uses updates through the old API, which means it cannot receive new updates for itself and for mods that depend on it. It seems to me that it’s worth mentioning this somewhere, at least on the mod page, because if I didn’t understand that updates weren’t coming, I might not update Beardlib for a long time

**Comment by Luffyyy** [comment]
It's written on the mod page on mws

**Reply by MrDogit** [reply]
Yeah, Im sorry, im too blind.
